pkg/server/team/handlers_test.go

```diff
@@ -297,3 +297,54 @@ func TestRemoveUserPermissionDenied(t *testing.T) {
 		t.Errorf("expected auth.ErrPermissionDenied, got %v", err)
 	}
 }
+
+func TestTeamRenameSuccess(t *testing.T) {
+	fake := NewFakeOperations()
+
+	expectedEmail := "teresa@luizalabs.com"
+	oldName := "teresa"
+	fake.(*FakeOperations).Storage[oldName] = &database.Team{Name: oldName, Email: expectedEmail}
+
+	s := NewService(fake)
+	ctx := context.WithValue(context.Background(), "user", &database.User{Email: "gopher", IsAdmin: true})
+
+	newName := "gophers"
+	req := &teampb.RenameRequest{OldName: oldName, NewName: newName}
+	if _, err := s.Rename(ctx, req); err != nil {
+		t.Fatal("Got error renaming team:", err)
+	}
+
+	oldTeam := fake.(*FakeOperations).Storage[newName]
```

Comment:
oldTeam -> newTeam ?

Reply:
no, the team is the same, that method only changes the name.